gef-ui/server/controllers/return-to-maker/index.js

```diff
@@ -0,0 +1,66 @@
+import {
+  getApplication, getUserDetails, updateApplication, setApplicationStatus,
+} from '../../services/api';
+import { validationErrorHandler } from '../../utils/helpers';
+import { PROGRESS } from '../../../constants';
+
+export const MAX_COMMENT_LENGTH = 400;
+
+export const getReturnToMaker = async (req, res) => {
+  const { params } = req;
+  const { applicationId } = params;
+  const { status } = await getApplication(applicationId);
+
+  if (status !== PROGRESS.BANK_CHECK) {
+    console.log('Incorrect status or permissions, redirecting to dashboard');
+    return res.redirect('/dashboard/gef/');
```

Comment:
Should this maybe go to the deal/application route instead?

Reply:
Can do - nothing specified in the ticket and was trying to avoid issue with too many redirects (this to application to dashboard or wherever)

Reply:
No biggie, just an observation - suppose this is a bigger piece of UX to do in the future e.g maybe nice set of error pages like "you don't have access", "unable to do this" etc